Makefile

```diff
@@ -146,13 +146,16 @@ check-certs: # Check cert subject
 	openssl x509 -noout -in /tmp/cert.pem --subject
 
 ##@ Build
-build: generate fmt vet imports ## Build manager binary.
+pre-build: ## Generate code, format it and organize imports before executing build
+pre-build: generate fmt imports vet
+
+build: pre-build ## Build manager binary.
 	go build -mod=vendor -o bin/manager main.go
 
-fast-build: generate fmt vet ## Fast build manager binary for local dev.
+fast-build: generate ## Fast build manager binary for local dev.
```

Comment:
@eloycoto changed this a bit to perform minimal steps for compilation and building the binary.
PTAL If you think it should be removed or changed.

Reply:
it's ok, it's ok :-)